### ISSUE TITLE
Audit #1 – General Overview when logged in – Checkboxes have no focus style. Buttons in files-list-row-actions-dropdown have no focus style

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4339]
Connected to Audit 1, see #4300 
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
- Checkboxes have no focus style. 
- All buttons in files-list-row-actions-dropdown have no focus style

# Remediation
Apply focus styles

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
